[ENH] Describe Inheritance Principle in Common Principles (#1807)
bids-standard/bids-specification: src/common-principles.md

## The Inheritance Principle

In some circumstances, there can be multiple data files for which
all or a subset of the relevant metadata is precisely equivalent.
Where this occurs,
it may be preferable to define those metadata *only once*,
and be placed on the filesystem in such a way that those files
are deemed to be *applicable* to each relevant data file individually,
but *not* be erroneously associated with other data files
to which the metadata contained within are not applicable.

> The Inheritance Principle defines a systematized set of rules by which

Reviewer: ```suggestion
The Inheritance Principle defines a systematized set of rules
```
to make it only 2 `which`, 3 of which cause my mental reset in parsing the sentence.

to determine which metadata files to associate with which data files.

> Further, due to the prospect of an individual data file having

Reviewer 2: I'm not a native speaker, but I would prefer a shorter sentence:

_Further, because multiple metadata files may apply to an individual data, the Principle defines the *order of precedence* of such metadata files contents._

Author: Adopted in 8154a6b648878634b6581e25bdd98336aca70c61.

> Further, due to the prospect of an individual data file having
> multiple metadata files applicable to it,
> the Principle defines the *order of precedence* with which such
> metadata file contents should be considered.

Reviewer 3: Adding @DimitriPapadopoulos's suggestion.

```suggestion
Further, because multiple metadata files may apply to an individual data file,
the Principle defines the *order of precedence* of such metadata files contents.
```
